### ISSUE TITLE
Attempt to parse HTTP_VIA as an IP fails

### ISSUE THREAD

**Issue by yourcelf** [un33k/python-ipware#13]
The `Via` HTTP header usually does not contain an IP address.  Per MDN, the expected syntax of that header's value is:
```
Via: [ <protocol-name> "/" ] <protocol-version> <host> [ ":" <port> ]
Via: [ <protocol-name> "/" ] <protocol-version> <pseudonym>
```
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Via#directives)

Examples of valid `Via` header values include:
```
Via: 1.1 vegur
Via: HTTP/1.1 GWA
Via: 1.0 fred, 1.1 p.example.net
```
None of those are IP's - and `python-ipware` v2.0.0 fails to parse e.g. `Via: 1.1 google` with this exception:
```
AddressValueError: Expected 4 octets in '1.1 google'
  File "python_ipware/python_ipware.py", line 106, in get_ip_object
    ip = ipaddress.IPv4Address(ipv4)
  File "ipaddress.py", line 1304, in __init__
    self._ip = self._ip_int_from_string(addr_str)
  File "ipaddress.py", line 1191, in _ip_int_from_string
    raise AddressValueError("Expected 4 octets in %r" % ip_str)
```

As a workaround, clients could override the precedence list to omit `HTTP_VIA` as an included value.  Given that this header typically does not contain an IP address, it may be preferable to omit it by default, and require clients that expect it to contain an IP address to provide it in their precedence list explicitly.

**Comment by un33k** [comment]
the code throws an exception in google's ipaddress package, but it is handled via two exception handling in python-ipware, and it skips to the next header. The only thing one should see is an exception log in the log files. 

Unless you have a log indicating that python-ipware fails to skip HTTP_VIA gracefully.  Let me know.

**Comment by yourcelf** [comment]
You're right, this is raised by python's built-in `ipaddress.py`, and then surfaced by `logging.exception(...)`, and it looks like control proceeds normally after that.  We saw this surface in a production environment where we typically treat the ERROR level logs and above as an indication of an exceptional error, and do things like publish them to Sentry and notify on-call engineers.  We never expect to see ERROR level logs as a normal fall-through control-flow event.

I'd still advocate form removing `HTTP_VIA` as a default header, since none of the forms of recommended syntax will result in a valid IP. Only clients providing a "malformed" value for `Via` will have an IP parsable using the current parser (they would need to omit the `<protocol-version>` part, and provide an IP as `<host>`).

I'd also advocate for reducing the logging severity to `logging.info()` -- if you want traceback info included, could use `logging.info(msg, exc_info=True)`, or at worst, `logging.warning(...)`.  That would play nicer with typical interpretation of logging levels for a control-flow event like this.

Either way: we've resolved this locally by providing a customized precedence list. Thanks for your work on this library.
